src/main/native/cpp/view/Analyzer.cpp

```diff
@@ -199,6 +199,11 @@ void Analyzer::Display() {
       }
 
       ShowGain("r-squared", &m_rSquared);
+      ShowGain("Sim RMSE", m_plot.GetRMSE());
+      CreateTooltip(
+          "This is the Root Mean Squared Error (RMSE) of the Simulation "
```

Comment:
Remove "This is" (capitalize The) and no need to capitalize the s in simulation.